### ISSUE TITLE
Autostructify works on sphinx 1.6.7 but fails on 1.7.4 with 'character_level_inline_markup' AttributeError

### ISSUE THREAD

**Issue by maxgrenderjones** [readthedocs/recommonmark#104]
- python 2.7.14
 - sphinx 1.7.4
 - recommonmark 0.4.0

```
def setup(app):
    app.add_config_value('recommonmark_config', {
            'enable_auto_doc_ref': True,
            'enable_math': True,
            'enable_inline_math': True,
            'enable_eval_rst': True
            }, True)
   #  app.add_transform(AutoStructify)
```
Runs, but doesn't do anything special with `eval_rst` directive on a fenced code block
```
def setup(app):
    app.add_config_value('recommonmark_config', {
            'enable_auto_doc_ref': True,
            'enable_math': True,
            'enable_inline_math': True,
            'enable_eval_rst': True
            }, True)
    app.add_transform(AutoStructify)
```
Fails with
```
reading sources... [100%] docs/sphinx
Exception occurred:
  File "/usr/local/lib/python2.7/site-packages/docutils/parsers/rst/states.py", line 470, in init_customizations
    if settings.character_level_inline_markup:
AttributeError: Values instance has no attribute 'character_level_inline_markup'
```

The text I'm trying to render:

    Tl;dr? If you want to include a table, you can. Thus:

        ```eval_rst
        +------------+------------+-----------+ 
        | Header 1   | Header 2   | Header 3  | 
        +============+============+===========+ 
        | body row 1 | column 2   | column 3  | 
        +------------+------------+-----------+ 
        | body row 2 | Cells may span columns.| 
        +------------+------------+-----------+ 
        ```

    renders as:

    ```eval_rst
    +------------+------------+-----------+ 
    | Header 1   | Header 2   | Header 3  | 
    +============+============+===========+ 
    | body row 1 | column 2   | column 3  | 
    +------------+------------+-----------+ 
    | body row 2 | Cells may span columns.| 
    +------------+------------+-----------+ 
    ```

It seems this kind of error has been fixed before, see https://github.com/sphinx-doc/sphinx/issues/4493

EDIT: when using sphinx 1.6.7 the behaviour is as desired

**Comment by asmeurer** [comment]
Updating docutils to 0.14 makes this error go away for me.

**Comment by arfon** [comment]
Just confirming that creating a `requirements.txt` file with `docutils==0.14` specified fixed this for me too.

**Comment by WasabiFan** [comment]
I also saw this error; I can confirm that @arfon's workaround solver the problem for me.
